doc-content/kogito-docs/src/main/asciidoc/drl/con_drl-rule-units.adoc

```diff
@@ -81,7 +81,7 @@ To define DRL rules that pattern-match against `Temperature` values, the example
 
 The `DataStream` data source is an append-only store for incoming data, similar to a queue. This type of data source is logical for both sources in this example because the temperature data is coming from an external source (the sensor) and the service publishes the alerts externally as they are produced.
 
-The example service uses the following `MonitoringService.drl` file in the same `src/main/resources/org/acme` folder of the {PRODUCT} project to declare the rule unit with the associated data sources and define the rules:
+The example service uses the following `MonitoringService.drl` file in the same `src/main/resources/com/acme` folder of the {PRODUCT} project to declare the rule unit with the associated data sources and define the rules:
```

Comment:
Path has to be coherent with the package name.

Reply:
Oops, thanks.